src/core/issues-notes.js

```diff
@@ -141,32 +141,36 @@ export function run(conf, doc, cb) {
     issueBase = conf.issueBase;
   if ($ins.length) {
     if (conf.githubAPI) {
-      (async function(githubAPI) {
-        try {
-          let json = await ghFetch(githubAPI);
-          issueBase = issueBase || json.html_url + "/issues/";
-          let issues = fetchIndex(json.issues_url, {
-            // Get back HTML content instead of markdown
-            // See: https://developer.github.com/v3/media/
-            Accept: "application/vnd.github.v3.html+json",
-          });
-          
-          issues.forEach(function(issue) {
-            ghIssues[issue.number] = issue;
-          });
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        } catch(err) {
-          pub("error", err.message);
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        }
-      })(conf.githubAPI);
+        fetchFunction(conf.githubAPI);
     } else {
       handleIssues($ins, ghIssues, issueBase);
       cb();
     }
   } else {
     cb();
   }
-}
\ No newline at end of file
+}
+
+async function fetchFunction(githubAPI) {
+  try {
+    let json = await ghFetch(githubAPI);
+    issueBase = issueBase || json.html_url + "/issues/";
+    let issues = await fetchIndex(json.issues_url, {
+    // Get back HTML content instead of markdown
+    // See: https://developer.github.com/v3/media/
+    Accept: "application/vnd.github.v3.html+json",
+  });  
+    
+    issues.forEach(function(issue) {
+      ghIssues[issue.number] = issue;
+    }); 
+    handleIssues($ins, ghIssues, issueBase);
+    cb();
+  
+  } catch(err) {
+      pub("error", err.message);
+      handleIssues($ins, ghIssues, issueBase);
+      cb();
```

Comment:
Not cb() is no longer in scope

Reply:
Err, “note”

Reply:
Thanks for noticing that! I didn't realize I'd put the function outside run().